larray/io/excel.py

```diff
@@ -171,12 +173,16 @@ def _ipython_key_completions_(self):
             return list(self.sheet_names())
 
         def __getitem__(self, key):
+            if isinstance(key, Group):
```

Comment:
I think it would be better to create an helper function "sanitize_excel_sheet_name" or something like that than copy-pasting the same code all over the place.

Reply:
"translate_excel_sheet_name"?